docs/contributing/docs-contributions.md

```diff
@@ -94,6 +94,13 @@ After going through the [development setup instructions](/contributing/setting-u
 - View the changes in your browser at `http://localhost:8000`.
 - Commit your changes and [submit a pull request](/contributing/how-to-open-a-pull-request/)!
 
+## Changing headers
+
+It can be necessary to change a heading within the docs. It's important to note that headers automatically generate links with a corresponding URL that can be deeplinked from elsewhere on the site. When changing a header be sure to point all corresponding links to the new URL.
```

Comment:
```suggestion
It can be necessary to change a heading within the docs. It's important to note that headers automatically generate links with a corresponding URL that can be deep-linked from elsewhere on the site. When changing a header, be sure to point all corresponding links to the new URL. Here are some workflow tips:
```